antora/docs/modules/ROOT/templates/_rule_links.hbs

```diff
@@ -0,0 +1,6 @@
+Rules included:
+
+{{#each this}}
+// Fixme: This link wouldn't work if used for pipeline rule collections
```

Comment:
Just for my understanding, this is because pipeline rules don't have collections? We don't use this for pipeline rules from what I can see, only in `policy_configuration.hbs`, so do we care? Should this be:

```suggestion
{{! This partial requires rule collection input }}
```

Reply:
It's because the link hard codes the `release_policy.adoc` file name. If we ever did have collections for the pipeline rules, (which is a reasonable thing we might want to do), then link would be broken. (Actually I just thought of a way to fix this, maybe I'll try it.)

Reply:
Fixed in https://github.com/hacbs-contract/ec-policies/pull/295 .